Doc/whatsnew/3.7.rst

```diff
@@ -466,6 +466,10 @@ Deprecated
 
 - The :mod:`macpath` is now deprecated and will be removed in Python 3.8.
 
+- Deprecated using :term:`bytes-like objects <bytes-like object>` except
```

Comment:
maybe this part should be moved to 3.8/rst?

Reply:
Indeed.